common/app/views/fragments/amp/stylesheets/glabs.scala.html

```diff
@@ -0,0 +1,136 @@
+.paid-content--advertisement-feature {
+    background-color: #d9d9d9;
+}
+
+.paid-content--advertisement-feature .content__meta-container:not(.content__meta-container--no-byline) {
+    margin: 0;
+}
+
+.badge--alt {
+    font-size: 14px;
+}
+
+.badge__link {
+    display: block;
+}
+
+.paidfor-band {
+    background: #69d1ca;
+    width: 100%;
+}
+
+.paidfor-meta,
+.paidfor-meta__label,
+.paidfor-label {
+    display: flex;
+}
+
+.paidfor-input {
+    display: none;
+}
+
+.paidfor-input:not(checked) ~ .popup--paidfor {
```

Comment:
Yeah perhaps in some browsers, but that is not standards compliant. In more zealous browsers that may be interpreted as an element selector, like `:not(div)`.